### ISSUE TITLE
Align EasySOA-registry on 5.6

### ISSUE THREAD

**Issue by tiry** [easysoa/EasySOA#87]
## 5.5.0 and 5.6-SNAPSHOT Alignment

I can test the alignment on 5.5, this should be easy and would prepare the work for 5.6-SNAPSHOT if I need to introduce new stuffs in Nuxeo for EasySOA.

5.6-SNAPSHOTs are available on our maven repo and are updated normally every night if all integration and non-regression tests are ok.

=> from the EasySOA project perspective this is not a big risk and it opens the way for doing infrastructure changes in the platform that directly benefit to EasySOA
## Packaging

I can create a MarketPlace package so that you can simply install EasySOA in a bare Nuxeo from the Admin center.

**Comment by mdutoo** [comment]
OK to test alignment. Then **TODO mkalam** move to depending on Nuxeo nightly snapshots, and update the release procedure.

About packaging : it would be cool and a quick win, but since it's packaging has to be first thought out in terms of project deliverables, and then what functional perimeter it will (can) provide, what it can't, and what the remaining is useful to and for which people (e.g. Nuxeo users ? SOA documentation ?).

**Comment by tiry** [comment]
The marketplace package is just a way to be able to install EasySOA registry in one click from the Admin Center UI from Nuxeo.
It's a replacement of the current "copy libs and plugins in Nuxeo and restart".

**Comment by mdutoo** [comment]
Yes but I was wondering up to where this will provide a working, functional EasySOA...

**Comment by mkalam-alami** [comment]
I'm starting to align our work with 5.5, since most of the work will probably be to update the overriden templates. However it's seems like there's no 5.6-SNAPSHOT bundles on your repository yet (https://maven.nuxeo.org/nexus/content/groups/public-snapshot/org/nuxeo/nuxeo-ecm/).

**Comment by tiry** [comment]
Wait for us to clean up the template override before upgrading to 5.5 : this will be more efficient !

**Comment by mkalam-alami** [comment]
Yeah you're right, I'll keep the work on hold then.

**Comment by tiry** [comment]
Can you propose me (via mail) a date where we can do the initial cleanup together.

**Comment by tiry** [comment]
Stated cleanup work - will push on a branch when ready.

**Comment by tiry** [comment]
## document_view.xhtml / easysoa_view.xhtml

=> added easysoa_view in a custom summary layout

=> bind this summary layout for EAsySOA types

## content_view.xhtml

defined a new EasySOA Content Tab and completly reuse existing template (no override)

## content_view_search_layout.xhtml / content_view_result_layout_selector.xhtml

Just remove the override, I think it's too bad to remove features.

As explained this could be done via configuration, but rather that removing all options vis configuration, I would prefer to do the upgrade and be sure you don't prefer to configure the features rather than removing them.

## Pom Alignement on 5.5

I did the pom alignement to be able to complete the template override removal and leveral the Themes flavors + new tab system.

I will need to spend some time with Julien to cleanup the maven config and we will probably need some history from Marwan.
